web/src/components/header/Header.jsx

```diff
@@ -33,31 +38,103 @@ const Header = () => {
 
   return (
     <div>
-      <Navbar
-        collapseOnSelect
-        expand="md"
-        fixed="top"
-        style={{ transition: '0.5s ease', backgroundColor: navBackground ? 'black' : 'transparent' }}
-      >
-        <Container>
-          <Navbar.Brand href="#home">
-            <div className="d-flex justify-content-between align-items-center">
-              <img alt="Poro logo" src={Poro} width="50" height="50" className="d-inline-block align-top" />
-              <div className="text-white">Poro</div>
-            </div>
-          </Navbar.Brand>
-          <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+      <Default>
+        <div>
+          <Navbar
+            collapseOnSelect
+            expand="md"
+            fixed="top"
+            style={{ transition: '0.5s ease', backgroundColor: navBackground ? '#2C394B' : 'transparent' }}
+          >
+            <Container>
+              <Navbar.Brand href="#home">
+                <div className="d-flex justify-content-between align-items-center">
+                  <a href="/">
+                    <img alt="Poro logo" src={Poro} width="50" height="50" className="d-inline-block align-top" />
+                  </a>
+                  <a href="/">
+                    <div className="text-white">Poro</div>
+                  </a>
+                </div>
+              </Navbar.Brand>
+              <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+
+              <Navbar.Collapse id="responsive-navbar-nav">
+                <Nav className="ml-auto">
+                  <Nav.Link
+                    onClick={() =>
+                      scroller.scrollTo('home', {
+                        smooth: true,
+                        duration: 500,
+                      })
+                    }
+                  >
+                    Home
+                  </Nav.Link>
+                  <Nav.Link
+                    onClick={() =>
+                      scroller.scrollTo('about', {
+                        smooth: true,
+                        offset: -70,
+                        duration: 500,
+                      })
+                    }
+                  >
+                    About
+                  </Nav.Link>
+                  <Nav.Link
+                    onClick={() =>
+                      scroller.scrollTo('commands', {
+                        smooth: true,
+                        offset: -70,
+                        duration: 500,
+                      })
+                    }
+                  >
+                    Commands
+                  </Nav.Link>
+                  {navBackground ? (
+                    <Nav.Link className="get-porobot" onClick={() => window.open(ADD_URL, '_blank')}>
+                      Get PoroBot
+                    </Nav.Link>
+                  ) : null}
+                </Nav>
+              </Navbar.Collapse>
+            </Container>
+          </Navbar>
+        </div>
+      </Default>
```

Comment:
If you are going to use bootstrap <Default> and <Mobile> components, it's better to put these out into separate components. 
eg; 
create new; HeaderDesktop.jsx and HeaderMobile.jsx and this should just import those components